zebra/rib.h

```diff
@@ -598,6 +598,8 @@ DECLARE_HOOK(rib_shutdown, (struct route_node * rn), (rn));
  */
 static inline struct nexthop_group *rib_get_fib_nhg(struct route_entry *re)
 {
+	if (re == NULL)
```

Comment:
I would prefer that we figure out why a call to rib-get_fib_nhg is even possible with a null pointer instead of this fix